manoseimas/settings/development.py

```diff
@@ -38,6 +38,14 @@
 
 INSTALLED_APPS += (
     'debug_toolbar',
+    'webpack_loader',
 )
 
 DATABASES['default']['NAME'] = 'manoseimas'
+
+WEBPACK_LOADER = {
```

Comment:
Hmm, I think then I should also have two webpack configs, one for prod, other for dev. Dev one has hot reload and is used with webpack-dev-server which is building into memory, not into filesystem.

Am I right?